frontend/__tests__/components/catalog.spec.tsx

```diff
@@ -2,9 +2,9 @@ import * as React from 'react';
 import * as _ from 'lodash-es';
 import { mount, ReactWrapper } from 'enzyme';
 
-import { CatalogTile } from '../../node_modules/patternfly-react-extensions/dist/js/components/CatalogTile';
-import { VerticalTabsTab } from '../../node_modules/patternfly-react-extensions/dist/js/components/VerticalTabs';
-import { FilterSidePanel } from '../../node_modules/patternfly-react-extensions/dist/js/components/FilterSidePanel';
+import { CatalogTile } from '@patternfly/react-catalog-view-extension/dist/js/components/CatalogTile';
```

Comment:
Do you have to do the ugly `/dist/js/components/CatalogTile` or are we not properly exporting `CatalogTile`, `VerticalTabsTab`, and `FilterSidePanelCategoryItem` upstream?

Reply:
This has been fixed.

Reply:
🏅